src/events/default-event-dispatcher.ts

```diff
@@ -177,8 +178,8 @@ export function newDefaultEventDispatcher(
   batchSize: number = DEFAULT_EVENT_DISPATCHER_BATCH_SIZE,
   config: Omit<EventDispatcherConfig, 'ingestionUrl' | 'sdkKey'> = DEFAULT_EVENT_DISPATCHER_CONFIG,
 ): EventDispatcher {
-  const sdkKeyDecoder = new SdkKeyDecoder();
-  const ingestionUrl = sdkKeyDecoder.decodeEventIngestionUrl(sdkKey);
+  const apiEndpointsHelper = new ApiEndpoints({ sdkTokenDecoder: new SdkKeyDecoder(sdkKey) });
```

Comment:
This needs a `defaultUrl` to avoid users over-writing the subdomain?

Reply:
Updated the docs and added a helper to clarify that this method is different from the other endpoints.